app/src/main/res/layout/listitem_weapon.xml

```diff
@@ -3,14 +3,19 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:padding="@dimen/padding_large">
+    android:layout_height="120dp"
```

Comment:
I think this is already a listitem height in dimens.xml. if it isn't then make it. Might end up being `xxLarge` or something.